server/webhooks/registration-parser.js

```diff
@@ -0,0 +1,84 @@
+"use strict";
+import 'babel-register';
+import _ from 'lodash'
```

Comment:
* Semicolon after statement
* We want to avoid importing all of lodash if possible.  Instead use named imports: example if I only need the extend function (`_.extend`) then i would do
  
  ```
  import { extend } from 'lodash';
  ```